### ISSUE TITLE
[low] Chrome: Company Information:  Incorporation date section content is overlapped.

### ISSUE THREAD

**Issue by Stutikuls** [Expensify/App#8383]
## Action Performed:
1. Using JAWS+ Chrome, open URL staging.new.expensify.com
2. Navigate to Setting using Tab key and press Enter to select it.
3. Navigate and select any workspace using Enter & Tab key.
4. Navigate to the connect bank account using Tab key and press Enter to select it.
5. Navigate to connect Manually using Tab key and press Enter to select it.
6. Navigate to Save & Continue button using Tab key and press Enter to select it from Connect Bank Account screen.
7. Verify the Incorporation date section content. 

## Expected Result:
 Incorporation date section content is not get overlapped.

## Actual Result:
 Incorporation date section content is getting overlapped.

**Other Occurrences:**
Same issue repro on PR https://github.com/Expensify/App/pull/8041

HTML Code - 
div class="css-1dbjc4n r-13awgt0 r-1pi2tsx r-12vffkv r-1wyyakw" style="flex-direction: row; -webkit-box-orient: horizontal; -webkit-box-direction: normal;"><input autocapitalize="sentences" autocomplete="on" autocorrect="on" dir="auto" spellcheck="true" type="date" class="css-11aywtz r-13awgt0 r-tz77pi r-13qz1uu expensify-datepicker" value="" max="2022-03-31" style="border-radius: 8px; border-width: 0px; box-shadow: none; color: rgb(11, 27, 52); font-family: GTAmericaExp-Regular; font-size: 15px; line-height: 20px; outline-width: 0px; padding: 23px 11px 8px;"></div

## Workaround:
Yes

## Area issue was found in:
Company Information 

## Failed WCAG checkpoints
Usable

## User impact:
Some people with vision, reading, and cognitive disabilities have trouble reading particular fonts or font weights or need more space between letters, words, or paragraphs to effectively read text content.

## Suggested resolution:
Fix this issue by allowing users to override text spacing styles by providing flexibility in the height and width of content containers. Some common techniques like :
1. Size containers to a have a value greater than the default width of the text. Small areas of text (such as navigation bars) need to have sufficient space within each box so that words and characters do not expand out of their containers.
 expand.

## Platform:
- Web

**Version Number:** v1.1.46-2
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**

![8176_content is overlaooed](https://user-images.githubusercontent.com/97600950/160967572-b60bb9fa-1270-4436-aa3c-e6b34c091989.png)
